EpikCord/__init__.py

```diff
@@ -823,8 +823,10 @@ async def handle_interaction(self, interaction):
 
                 for option in interaction.options:
                     options.append(option.get("value"))
-
-            return await command.callback(interaction, *options)
+            try:
+                return await command.callback(interaction, *options)
+            except Exception as e:
+                self.command_error(e, interaction=interaction)
```

Comment:
Make this a coro?

Reply:
Yeah, you made the function async, but not awaited it. Fix that and it looks good.